plugins/inputs/couchbase/couchbase.go

```diff
@@ -71,15 +89,26 @@ func (r *Couchbase) gatherServer(addr string, acc telegraf.Accumulator, pool *co
 		}
 		pool = &p
 	}
+
+	sanitizedAddress, err := sanitizeURI(addr)
+	if err != nil {
+		return err
+	}
+
+	if len(sanitizedAddress) <= 1 {
+		log.Printf("I! WARNING: Cluster address tag \"'%s'\" is too short.", sanitizedAddress)
```

Comment:
This line should be hit when server is empty, but you are right it should be filtered by configuration parser.